server/app/migrations/0003_alter_profiles_avatar_url.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 5.0.6 on 2024-07-17 09:49
+
+import app.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0002_alter_profiles_avatar_url'),
```

Comment:
It seems that you forget to add '0002_alter_profiles_avatar_url' to commit

Reply:
![image](https://github.com/user-attachments/assets/be057662-898f-40f8-83c4-7a639a4d06f2)
0002_alter_profile_avatar_url is here